edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Mqtt.Test/MqttUsernameParserTest.cs

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft. All rights reserved.
-namespace Microsoft.Azure.Devices.Edge.Hub.Core.Test
+namespace Microsoft.Azure.Devices.Edge.Hub.Mqtt.Test
 {
     using System.Collections.Generic;
-    using Microsoft.Azure.Devices.Edge.Hub.Mqtt;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
```

Comment:
This looks suspicious.

Reply:
This test file was in the wrong namespace (the namespace should match the folder name), probably due to copy/paste error, so I fixed it. Since I changed the namespace it belongs to, that impacted what other classes are visible to the file, so I had to update the `using` statement to match.